tests/unit/AddNameTranslation.spec.ts

```diff
@@ -31,163 +30,190 @@ describe('Add Name Translation component', () => {
     // Init Store
     store.state.stateModel.nameTranslations = []
 
-    wrapperFactory = (propsData) => {
-      return mount(AddNameTranslation, {
+    wrapperFactory = async (propsData: any) => {
+      const wrapper = mount(AddNameTranslation, {
         localVue,
         router,
         store,
         vuetify,
         propsData: { ...propsData }
       })
+      await Vue.nextTick()
+      return wrapper
```

Comment:
Most of the time, you need to wait for things to stabilize after mounting a component.